### ISSUE TITLE
Support for HTML entities

### ISSUE THREAD

**Issue by D-Bullock** [robert-j-engdahl/ngettext-wpf#27]
If one uses a html entity (say &quot; for a " sign) then the translation doesn't work - it can't find the string in the po file.

What happens is that the gettext extension receives the content with the &quot; converted into a ", ie if we were to use `{wpf:Gettext &quot;example&quot;}` the gettext extension would see "example"

I *think* the correct way to solve the problem is to change the C# MO parser to convert html entities, in the same way that the XAML parser is doing. The other solution would be to change the extractor to convert the HTML entities. 

To get around the issue, I'm currently doing a manual find/replace in the generated PO files

**Comment by robert-j-engdahl** [comment]
Yes, this should be fixed in the XAML internationalization process, i.e. `XGetText-Xaml.ps1`.

I think XAML is supposed to support all XML character entities.

Here is an example of something we might be able to use in solving this:

https://stackoverflow.com/a/10082285/2154774

Pull requests are welcome
